code/drasil-docLang/Drasil/Sections/Requirements.hs

```diff
@@ -1,5 +1,5 @@
 module Drasil.Sections.Requirements (fReqF, fullReqs, fullTables, inReq, inTable,
-  mkInputPropsTable, mkQRTuple, mkQRTupleRef, mkValsSourceTable, nfReqF, reqF) where
+  mkInputPropsTable, mkQRTuple, mkQRTupleRef, mkValsSourceTable, nfReqF, reqF, reqInputsUID) where
```

Comment:
Yea, I think there could be something better than just transporting the UIDs. I would use the `mkInputPropsTable` function directly, but it takes in a Referable for its second argument (even though the reference database doesn't need to know that information. 

I could replace the reqInputsUID with the actual reference for the table and pass that through. It would probably be less hacky and more obvious that it is being used as a reference rather than just having a UID.